ably/rest_client.go

```diff
@@ -92,27 +86,24 @@ func (c *RestClient) Stats(params *PaginateParams) (*PaginatedResource, error) {
 }
 
 func (c *RestClient) Get(path string, out interface{}) (*http.Response, error) {
-	req, err := http.NewRequest("GET", c.RestEndpoint+path, nil)
+	req, err := http.NewRequest("GET", c.options().restURL()+path, nil)
 	if err != nil {
 		return nil, err
 	}
 	req.Header.Set("Accept", "application/json")
-	req.SetBasicAuth(c.Auth.Options.Token, c.Auth.Options.Secret)
-	res, err := c.httpclient().Do(req)
-
+	// TODO(rjeczalik): add support for token auth
```

Comment:
I don't really like Trello in general... But ignoring that, I think that since this is an open source client library, the best is to place these things in our issues and add a label for `features` for transparency.